app/gui/src/project-view/components/GraphEditor/GraphNodeOutputPorts.vue

```diff
@@ -84,7 +85,7 @@ const outputPorts = computed((): PortData[] => {
 // === Interactivity ===
 
 const mouseOverOutput = ref<AstId>()
-const mouseOverCreateNodeFromPortButton = ref(false)
+const mouseOverCreateNodeFromPortButton = ref(0)
```

Comment:
Do I understand correctly that now it has some “default” value, to distinguish “false that was never changed” from “false that did change”? I think a comment would be useful here.